src/info.json

```diff
@@ -19,7 +19,7 @@
     ],
     "title": "Subspace Storage (Alpha)",
     "author": "Clusterio Team",
-    "homepage": "https://github.com/clusterio/factorioClusterio",
+    "homepage": "https://github.com/clusterio/subspace_storage",
     "description": "Allows storing items and fluids in a limitless shared subspace.  Warning: work in progress alpha.",
```

Comment:
Is subspace storage alpha? Seems pretty solidly tested now, and no new functionality is planned to my knowledge.

Reply:
I've had ideas, such as make it work without Clusterio for a single player world, make it use the new serializer so items retain their data, reimplement inventory import as an equipment, move the plugin code to this repository, do something more interesting about the restriction zone(s). That kind of stuff. There just doesn't seem to be enough time to do any of it :(.